packages/driver/src/cy/multi-domain/index.ts

```diff
@@ -161,8 +161,6 @@ export function addCommands (Commands, Cypress: Cypress.Cypress, cy: Cypress.cy,
               env: preprocessEnv(Cypress.env()),
             })
 
-            state('readyForMultiDomain', true)
```

Comment:
This has been completely removed, we now rely on timeouts for handling multi-domain errors.